akd_client/src/tests.rs

```diff
@@ -392,6 +407,7 @@ async fn test_history_proof_multiple_epochs() -> Result<(), AkdError> {
 }
 
 #[tokio::test]
+#[allow(dead_code)]
```

Comment:
Can we remove this?

Reply:
All `#[allow(dead_code)]`s should be removed after enabling the tests. I mentioned this in a few other places. @Jasleen1 said she would do a clean up before merging.